packages/calcite-design-tokens/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to `calcite-design-tokens`
+
+This document contains information about contributing to the `calcite-design-tokens` package. Please read the general [CONTRIBUTING](../../CONTRIBUTING.md) document first, which includes initial setup, commit message conventions, and more.
+
+## Project scripts
+
+### Build
+
+Build the tokens to platform asset formats like CSS, SCSS, and JavaScript.
+
+`npm --workspace=packages/calcite-design-tokens run build`
+
+## Understanding Token Files
```

Comment:
Nitpick: We use sentence case for our doc headings, e.g. `Understanding token files`